command/command.go

```diff
@@ -206,6 +214,10 @@ var commandsHelp = []string{"To get tool version",
 	"This command will initiate a self RBT transfer",
 	"This command will unpledge all the pledged tokens",
 	"This command will unpledge all PoW based pledge tokens and drop the unpledgequeue table",
+	"This command will create FT",
```

Comment:
Orders are mismatched. Confirm with -help command if the description are coming as expected

Reply:
Order updated. Confirmed with -h flag.